_data/cloud.yml

```diff
@@ -10,8 +10,7 @@ websites:
       url: https://www.ctl.io/appfog/
       img: appfog.png
       bch: No
-      twitter: appfog
-      facebook: CenturyLinkCloud
+      twitter: centurylinkcld
```

Comment:
I think there's a problem here. Should be 'appfog' for Twitter username, no?

Reply:
nope, I went through these, no twitter account should be listed.

Reply:
Fair enough, but there still is an incorrect Twitter handle (centurylinkcld) for appFog. I have the branch local and can push a quick change to remove the Twitter handle completely if you wish.

Reply:
its one and the same. I verified it myself because no one uses the handle centurylinkcloud anymore

Reply:
Sorry, I'm stupid. It appears that 'centurylinkcld' is the parent company's Twitter handle. Still though, should this listing not use @appFog on Twitter?

Reply:
do they use that handle? I was tempted to remove it entirely as we have Centruylink in another section anyways and it doesnt look like this product is going anywhere

Reply:
They do, I checked, and for consistency's sake, I suggest we use it for appFog. I'll push a quick commit through, then approve this PR.